external-testing/README.md

```diff
@@ -0,0 +1,60 @@
+# External Testing
+
+Let's talk about testing process against external resources. What does that
+mean? Well, talking to GitHub API, COPR API, running `fedpkg`, cloning
+repositories...
+
+
+## Use cases
+
+* Interact with an external API (such as GitHub, COPR, Pagure, FAS, k8s)
+  * Authenticated vs. unauthenticated calls.
+* Clone a git repository.
+* Construct a git repository
+* Run a command to perform an action.
+  * Changes happening only locally.
+  * RO access to a remote service.
+  * R/W access to a remote service.
+* Combination of multiple use cases into a single test case.
+
+
+## Solutions
+
+* flexmock
+
+* pytest fixtures
+
+* requre
+
+* VCRpy
+
+
+## VCRpy vs. requre
+
+The section you are craving to read. It's opinionated and not objective.
+
+| A feature | VCRpy | requre |
+|-----------|-------|--------|
+| Record & replay for python requests | ✓ | ✓ |
```

Comment:
Compatibility with `vcr` cassettes? Using some kind of standard can be beneficial. (If there is be some tool e.g. for postprocessing/visualisation/... it will probably be for some standard format.)

[Betamax](https://github.com/betamaxpy/betamax) should be compatible. Not sure about `vcrpy`.

Reply:
what?

Reply:
```python
if what == "supprise":
     return "Just read it in the betamax readme. If there are multiple recording tools that support the same format, we can switch between them more easily."
elif what == "I don't understand.":
    return "The response format can be shared between multiple implementations. E.g. with original ruby `vcr`."
else:
    return "What do you mean?"
```

Reply:
https://github.com/orgs/betamaxpy/people if we want some redhater maintaining the project...;)

Reply:
they are simple yaml files, even if they deviated from the standard, we should be able to transform them - I'd say it should be easily feasible to e.g. transform requre response files to vcr